.github/dependabot.yml

```
@@ -10,6 +10,53 @@ updates:
     directory: "/"
     # The maximum number of pull requests for version updates
     open-pull-requests-limit: 10
+    groups:
+      radix-ui:
+        patterns:
+          - "@radix-ui/*"
+      graphql:
+        patterns:
+          - "graphql"
+          - "@graphql-codegen/*"
+          - "@0no-co/graphqlsp"
```

```suggestion
      graphql-codegen:
        patterns:
          - "@graphql-codegen/*"
```

I suggest that you reduce this to a single group for `graphql-codegen`. The other packages do not have a direct relationship or dependency with each other - apart from the fact that they all deal with GraphQL.

The `@graphql-codegen/*` packages affect generated code so it makes sense to group them so we only have to re-generate code once but this is not the case with the other packages.

In other words:
- If the `graphql` page is updated nothing is likely to break if `@0no-co/graphqlsp` is not also updated.
 - If `@graphql-codegen/client-preset` is updated and `@graphql-codegen/introspection` there might be a dependency and we will like want to update both at the same time.